src/app.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
```

Comment:
please run prettier on all files

Reply:
If I put double quotes instead of single quotes to the import path, I get an that says to use single quotes.

Reply:
from eslint?

Reply:
eslint?!?!